RabbitMq/1.0.0/installer-manifest.json

```diff
@@ -0,0 +1,8 @@
+{
```

Comment:
This file is the recipe for the AssetExternalStorage (AssetStorage) module.

Reply:
Fixed